tests/js/range/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -26,6 +26,78 @@ a = [, , , , , , , a];
 ================================================================================
 `;
 
+exports[`boundary.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+rangeEnd: 26
+rangeStart: 13
+     |                                                                                | printWidth
+=====================================input======================================
+> 1 | foo = 1.0000;bar = 1.0000;baz=1.0000;
+    |              ^^^^^^^^^^^^^
+  2 | // The range will be 13~26
+  3 | // \`foo\` ends at 13, should not format
+  4 | // \`bar\` ends at 26, should format
+  5 | 
+=====================================output=====================================
+foo = 1.0000;bar = 1.0;baz=1.0000;
+// The range will be 13~26
+// \`foo\` ends at 13, should not format
+// \`bar\` ends at 26, should format
+
+================================================================================
+`;
+
+exports[`boundary-2.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+rangeEnd: 39
+rangeStart: 22
+     |                                                                                | printWidth
+=====================================input======================================
+  1 | function a(
+  2 | ){
+  3 | 
+  4 | a (
+> 5 | );                 b (
+    |   ^^^^^^^^^^^^^^^^^
+  6 | );
+  7 | 
+  8 | }
+  9 | 
+=====================================output=====================================
+function a(
+){
+  a();
+  b();
+}
+
+================================================================================
+`;
+
+exports[`boundary-3.js format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+rangeEnd: 23
+rangeStart: 6
+     |                                                                                | printWidth
+=====================================input======================================
+  1 | a (
+> 2 | );                 b (
+    |   ^^^^^^^^^^^^^^^^^
+  3 | );
+  4 | 
+=====================================output=====================================
+a (
+);                 b (
+);
```

Comment:
I think I got it correct now. This one didn't get formatted is another issue I was going to fix after this one, actually the same as #9673.